static/src/stylesheets/amp/_header.scss

```diff
@@ -1,334 +1,16 @@
-    // See PR: https://github.com/guardian/frontend/pull/13905
-    display: flex;
-    flex-wrap: wrap;
-    overflow: hidden;
-
-    @include mq($until: mobileMedium) {
-        padding-top: $gs-baseline / 3;
-    }
-
-    @include mq(mobileMedium) {
-        padding-top: $gs-baseline;
-    }
-}
-
-.main-navigation__item {
-    // Override inherited ul from user agent stylesheet
-    list-style: none;
-    color: $news-support-1;
-    width: 100%;
-}
-
-//This variable is only used in open menu styling
-$navigation-horizontal-padding: $gs-gutter * 2 + $gs-gutter / 2;
-
-.main-navigation__item__button {
-    position: relative;
-    padding-top: $gs-baseline / 4;
-    padding-left: $navigation-horizontal-padding;
-    padding-bottom: $gs-baseline;
-    cursor: pointer;
-    font-size: 1em;
-    font-weight: normal;
-    border: 0;
-    background-color: $guardian-brand-dark;
-}
-
-.main-menu__icon--chevron {
-    //optical alignment
-    margin-top: -.2em;
-
-    &:before {
-        content: '';
-        display: inline-block;
-        width: 8px;
-        height: 8px;
-        transform: rotate(45deg);
-        border: 2px solid $news-support-2;
-        border-top: 0;
-        border-left: 0;
-    }
-
-    section[expanded] & {
-        margin-top: .2em;
-
-        &:before {
-            transform: rotate(-135deg);
-        }
-    }
-}
-
-.main-menu__icon {
-    position: absolute;
-    left: $gs-gutter / 2;
-    text-align: center;
-    width: $navigation-horizontal-padding - $gs-gutter/2;
-}
-
-.main-menu__icon__svg {
-    fill: $news-support-2;
-    margin-top: -.05em;
-    width: 1em;
-    height: 1em;
-}
-
-.main-navigation__secondary {
-    color: #ffffff;
-    background-color: #003b5e;
-    padding-bottom: $gs-baseline / 2;
-
-    .main-menu__icon__svg {
-        margin-top: -.24em;
-        width: .8em;
-        height: .8em;
-    }
-}
-
-.navigation-border {
-    // Creates a seamless line when section has dark background
-    margin-top: -1px;
-    position: relative;
-
-    &:before {
-        content: '';
-        position: absolute;
-        top: 0;
-        left: $navigation-horizontal-padding;
-        right: 0;
-        height: 1px;
-        background: #003b5e;
-    }
-}
-
-.navigation-group li:last-child {
-    padding-bottom: $gs-baseline;
-}
-
-.navigation-group--grey {
-    background-color: $neutral-1;
-}
-
-.navigation-group__item {
-    // Override inherited ul from user agent stylesheet
-    list-style: none;
-    position: relative;
-    font-size: 1em;
-
-    a {
-        display: block;
-        padding-top: $gs-baseline / 1.5;
-        padding-left: $navigation-horizontal-padding;
-        padding-right: $gs-gutter;
-        padding-bottom: $gs-baseline / 1.5;
-        // Override a from user agent stylesheet
-        color: inherit;
-    }
-}
-
-.secondary-navigation {
-    font-size: .9em;
-    margin: 0;
-}
+@import '../layout/new-header/_variables';
```

Comment:
> I think we can share most of it - I'm trying to come up with something that takes the non-mobile related stuff out of the AMP include. All the rest is shareable I think.

Potentially, but we did simplify the design slightly for AMP as well

> Let's do not review this yet and go through it on Monday together. I think that's much easier.

Even then, this should be broken up into smaller changes.